lightning/src/chain/channelmonitor.rs

```diff
@@ -2331,6 +2354,7 @@ impl<Signer: EcdsaChannelSigner> ChannelMonitor<Signer> {
 						res.push(Balance::ClaimableAwaitingConfirmations {
 							amount_satoshis: prev_commitment.to_self_value_sat,
 							confirmation_height: conf_thresh,
+							source: BalanceSource::CounterpartyForceClosed,
```

Comment:
Should this be `HolderForceClosed`? Seems that we're comparing the txid to our previous holder commitment tx.

Reply:
This is true. I did misinterpret that field name and didn't check the comments.